mkdocs.yml

```diff
@@ -157,6 +157,7 @@ nav:
       - 'React JavaScript Tutorial in Emacs': tutorials/reactjs-tutorial.md
       - 'Configuring Emacs as a Clojure IDE': tutorials/clojure-guide.md
       - 'Debugging Clojurescript': tutorials/debugging-clojure-script.md
+      - 'Python, debbuging with poetry + pyenv': page/python-poetry-pyenv.md
```

Comment:
I'm a little bit confused here, this page doesn't exists on lsp-mode, but only on dap-mode right? I think you want to paste here the full URL link to dap-mode page?

Reply:
You're very correct, I forgot to push the actual URL... Is that better?